Documentation/contributing/development/dev_setup.rst

```diff
@@ -101,7 +101,7 @@ Cilium images:
   into the cluster.
 * ``make kind-image-operator``: Builds the Cilium Operator (generic) image only
   and loads it into the cluster.
-* ``make kind-image-debug``: Builds all Cilium images with optimizations
+* ``make kind-debug-agent``: Builds all Cilium images with optimizations
```

Comment:
there's `make kind-debug` too, which makes both agents and the operator available for debugging - should we use that throughout, or is there a reason you went for `kind-debug-agent`?

Reply:
Hmm, no particular reason, I just transcribed Joe's instructions in the cover letter of https://github.com/cilium/cilium/pull/21108. I'll present `kind-debug` as the main workflow, with `kind-debug-agent` as a subset for faster iteration cycles if rebuilding the operator is not necessary.